docker-compose.yml

```
@@ -19,6 +19,22 @@ services:
     environment:
       ME_CONFIG_MONGODB_ADMINUSERNAME: root
       ME_CONFIG_MONGODB_ADMINPASSWORD: example
-  # TODO: Add frontend webServer
+
+  webserver:
+    build:
+      context: webserver
+      dockerfile: Dockerfile
+    image: webserver:latest
+    restart: always
+    environment:
+      APP_PORT: 5000
+      MONGODB_DATABASE: weatherstation
+      MONGODB_USERNAME: root
+      MONGODB_PASSWORD: example
+      MONGODB_HOSTNAME: mongodb
```

```suggestion
      MONGODB_HOSTNAME: mongo
```
Kan se ut til at mongo er riktig siden servicesen til mongodb heter `mongo` i linje 4